modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc

```diff
@@ -9,7 +9,23 @@ Clusters using a restricted network must import the default must-gather image to
 
 .Procedure
 
-* Import the default must-gather image from your installation payload:
+. If you have not added your mirror registry's trusted CA to your cluster's image configuration object as part of the Cluster Samples Operator configuration, perform the following steps:
```

Comment:
Does _ installing/install_config/installing-restricted-networks-preparations.adoc_ need to be updated to _post_installation_configuration/cluster-tasks_?